### ISSUE TITLE
Wrong tags list I saw

### ISSUE THREAD

**Issue by KingSejong25** [Ninjdai1/letsdo-addon-compat#6]
![image](https://github.com/Ninjdai1/letsdo-addon-compat/assets/158523324/344c6e99-201f-4a61-8f27-2a7b71bfd17c)
![image](https://github.com/Ninjdai1/letsdo-addon-compat/assets/158523324/b7f78c20-80a8-447e-b9cb-fe638b67ccbb)
Chunk nugget(Delightful mod's food)
▶ Recipes used
- [Let's do] Meadow cauldron

=================================================

![image](https://github.com/Ninjdai1/letsdo-addon-compat/assets/158523324/d951d1c6-a46b-4d5a-bd16-ffa9475f00c3)
![image](https://github.com/Ninjdai1/letsdo-addon-compat/assets/158523324/f0520b61-e936-43a9-b971-d109a9753415)
**raw_pasta** item tag, a lot of pasta recipe's 
 ▶ Recipes used
- [Let's do] Meadow's cooking cauldron
- [Let's do] Candlelight's cooking pot
- [Let's do] Bakery's small cooking pot

Example picture's food is Crab pasta in Quark Delight.

=================================================

![image](https://github.com/Ninjdai1/letsdo-addon-compat/assets/158523324/9660d26c-5fff-4228-8e07-ffb8c696df1e)
![image](https://github.com/Ninjdai1/letsdo-addon-compat/assets/158523324/51d8b00e-edfd-4a47-ae02-c3ea8c7a3d98)
raw_rabbit tag in rabbit stew recipes
▶ Recipes used
 - [Let's do] Meadow's cooking cauldron
 - [Let's do] Candlelight's cooking pot
 - [Let's do] Bakery's small cooking pot

=================================================

![image](https://github.com/Ninjdai1/letsdo-addon-compat/assets/158523324/ade0744a-9b60-4d50-a9f3-9eb587b7fd62)
![image](https://github.com/Ninjdai1/letsdo-addon-compat/assets/158523324/9860f92b-120d-46dc-bcee-e34aee75b90e)
Coconut curry(Delightful mod's food)
▶ Recipes used
 - [let's do] Meadow cauldron

=================================================

![image](https://github.com/Ninjdai1/letsdo-addon-compat/assets/158523324/617a66e5-a7cc-4ddd-9352-29b51aa3a5e5)
![image](https://github.com/Ninjdai1/letsdo-addon-compat/assets/158523324/21ad03b7-e57e-4526-8913-2b584bbfc3a7)
![image](https://github.com/Ninjdai1/letsdo-addon-compat/assets/158523324/b2157639-338d-4aa7-89a9-f19688bf8542)
Lasagna([Let's do] Candlelight mod's food
▶ Recipes used
 - [Let's do] Meadow's cooking cauldron
 - [Let's do] Bakery's small cooking pot

=================================================

![image](https://github.com/Ninjdai1/letsdo-addon-compat/assets/158523324/d89afe1f-7b7f-469d-98c4-a9f5eb3726df)
![image](https://github.com/Ninjdai1/letsdo-addon-compat/assets/158523324/9f25760f-f298-416a-9f59-4bfa950c72ca)
Pineapple Fried rice(Pineapple Delight mod's food)
▶ Recipes used
 - [Let's do] Meadow's cooking cauldron
 - [Let's do] Candlelight's cooking pot
 - [Let's do] Bakery's small cooking pot

=================================================

![image](https://github.com/Ninjdai1/letsdo-addon-compat/assets/158523324/a68650b3-1670-4614-868a-99a9dbe2c4d3)
![image](https://github.com/Ninjdai1/letsdo-addon-compat/assets/158523324/6cdaa2be-de69-4a11-92b4-c27aef94b85e)
Phantom puff(Casualness Delight mod's food)
▶ Recipes used
 - [let's do] Meadow's cooking cauldron

P.S.) This mod is not in the list of compatible modes, but I added it because the recipe exists.

=================================================

![image](https://github.com/Ninjdai1/letsdo-addon-compat/assets/158523324/327a3ce2-d92b-4d57-92af-1d4608c39606)
![image](https://github.com/Ninjdai1/letsdo-addon-compat/assets/158523324/4621cef4-155e-4628-b04f-6965b21b3900)
Azalea Tea(Delightful mod's food)
▶ Recipes used
 - [Let's do] Meadow's cooking cauldron
 - [Let's do] Herbalbrews's Tea kettle

These are all I've seen.
Most errors occur in raw_pasta tag.

My English is not good.
Because I am Korean and this was helped by the translator. (a little!)
I'd appreciate it if you could check it out and tell me what you don't understand.
Thanks!

**Comment by Ninjdai1** [comment]
Hi,
Thanks for reporting all these !
Could you try the [development version](https://github.com/Ninjdai1/letsdo-addon-compat/releases/tag/latest) and confirm these issues still appear with it ?

Thanks ^^

**Comment by KingSejong25** [comment]
I checked it out.
There doesn't seem to be much difference.

Additionally, I found one more new problem.

![image](https://github.com/Ninjdai1/letsdo-addon-compat/assets/158523324/8d14ce00-1287-4b85-9775-d6d21d1e5e98)
![image](https://github.com/Ninjdai1/letsdo-addon-compat/assets/158523324/558ffa3c-6ad1-4238-903d-d4f8b2883de9)
Added by [Let's do] Meadow mod's cooked rabbit recipe
▶ Recipes used
- Farmer's Delight mod's Cooking pot 
- Miner's delight mod's Copper pot

**Comment by wasdspace** [comment]
![2024-06-27_20 23 23](https://github.com/Ninjdai1/letsdo-addon-compat/assets/85009668/901c8abc-b3bb-4a1c-9a47-9a2f6146c3d2)
![2024-06-27_20 23 32](https://github.com/Ninjdai1/letsdo-addon-compat/assets/85009668/c47d3ec5-0f29-4fd9-ad4e-ba3e80f3c8b7)
![2024-06-27_20 23 33](https://github.com/Ninjdai1/letsdo-addon-compat/assets/85009668/f858fbd6-859b-4cb0-9893-8db323ba565a)

**Comment by KingSejong25** [comment]
I don't know about **Create mod**.
Because I didn't use that mode.

The pictures above are all I've seen.

**Comment by wasdspace** [comment]
Yes but it shouldn't matter. To test I downloaded Farmers Delight, Create and Lets Do Series and Ninjdai1's mods

**Comment by Ninjdai1** [comment]
I mainly test on fabric, so I let quite a few tags error on forge slip
I'll fix all the reported ones, except `rottenleather:sweetened_chunk` as it's not a mod the compat addon impacts

**Comment by Ninjdai1** [comment]
> Additional informaltion on forge side, all the recipes use Tag seem to be corrupted. For example, if you craft recipes with water, the create's recipes would be {"amount": xx, "fluid": "minecraft:water", "nbt": {}}, but this mod uses "FluidTag:forge:water"

The FluidTag is used as that's what create uses aswell, I'll fix it

As for the forge: tags that don't work, can you check out the indev build and confirm the issues are still happening ?
Thanks for your reports, it really helps !

**Comment by KingSejong25** [comment]
Thanks!
But  `rottenleather:sweetened_chunk` is not the mode installed on my side either.
I don't know why there are this tag.

**Comment by druidicmage** [comment]
Not sure if anything here has been fixed, but on forge 1.20.1 (version 1.4.2) there are still issues
![image](https://github.com/Ninjdai1/letsdo-addon-compat/assets/74755923/62eddf1b-9376-4c6f-bf12-355d76acffbb)
![image](https://github.com/Ninjdai1/letsdo-addon-compat/assets/74755923/c43df89e-bc65-4147-9187-19747e9286c2)
![image](https://github.com/Ninjdai1/letsdo-addon-compat/assets/74755923/a5e3f23a-492b-4ef1-b165-5632980bb014)
